spec/features/users/index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page' do
+  before :each do
+    load_test_data
+  end
+
+  #  When a user visits the root path they should be on the landing page ('/') which includes:
+  #  Title of Application
+  #  Button to Create a New User
+  #  List of Existing Users which links to the users dashboard
+  #  Link to go back to the landing page (this link will be present at the top of all pages)
+
+  it 'displays a title' do
+    visit(root_path)
+    expect(page).to have_content('Viewing Party Lite')
+  end
+
+  it 'displays a button to create a new user' do
+    visit(root_path)
+    expect(page).to have_link('Create New User')
+  end
+
+  it 'displays a list of existing users' do
+    visit(root_path)
+    expect(page).to have_content(@user1.name)
+    expect(page).to have_content(@user1.email)
+    expect(page).to have_content(@user2.name)
+    expect(page).to have_content(@user2.email)
+  end
```

Comment:
Each user needs to link to their respective show page (change tests to `expect(page).to have_link(@user.name)` 
After merging, test should then be updated to `expect(page).to have_link(@user.name, href: user_path(@user))`